src/tests/resistance/handshake.rs

```diff
@@ -101,6 +111,137 @@ async fn r001_t2_HANDSHAKE_reject_if_server_too_long() {
     node.stop().unwrap();
 }
 
+/// Decide whether to enable node logs and tracing for synthetic nodes.
+#[derive(Clone, Copy)]
+enum Debug {
```

Comment:
I'm slightly worried about the name here.

Reply:
my logic was:
when some test is failing -> I need to debug that test which implies enabling logs/tracing
so I kinda put it under the `Debug On/Off` enum

what name would you prefer or have in mind?